### ISSUE TITLE
About inference code

### ISSUE THREAD

**Issue by KleinXin** [SSL92/hyperIQA#8]
I used some data of my own to train the model and want to calculate the score of images one by one.

During the trainning, I added some codes to calculate the l1-norm, such as the codes below

`for img, label in data:
            # Data.
            
            #img = torch.tensor(img.cuda())
            #label = torch.tensor(label.cuda(async=True))
            
            
            img = img.cuda().clone().detach()
            label = label.cuda(async=True).clone().detach()
            
            paras = self.model_hyper(img)
            model_target = models.TargetNet(paras).cuda()
            model_target.train(False)
            pred = model_target(paras['target_in_vec'])

            pred_scores.append(float(pred.item()))
            gt_scores = gt_scores + label.cpu().tolist()
        
        pred_scores_np = np.array(pred_scores,dtype=np.float32)
        gt_scores_np = np.array(gt_scores,dtype=np.float32)    
        l1_norm_test = np.absolute(pred_scores_np-gt_scores_np)
        l1_norm_test = np.sum(l1_norm_test)/len(l1_norm_test)`

**Comment by SSL92** [comment]
At inference stage, you should only load the parameters of hyper network and shouldn't load target network parameters, _i.e._

`model_target.load_state_dict(torch.load(args.pretrained_model_name_target))`

this line of code should be deleted. Since the parameters of the target network is generated from the hyper network adaptively, the code 'model_target = models.TargetNet(paras).cuda()' has already built a target network which has its own self-adaptive weight parameters.

Also, I noticed you have resized all the testing images to the size of 224x224, if you trained also with images resized to 224x224, it's OK, but if the training procedure follows the configuration in our origin code, _i.e._ randomly cropping 224x224 patches, it'll be better to use the same configuration to testing images, since scale consistency also influences model performance.

Hope this will help you : )
